components/chart/src/selection-area/selection-area.ts

```diff
@@ -794,26 +800,43 @@ export class DtChartSelectionArea implements AfterContentInit, OnDestroy {
 
     // handling of the overlay for the range
     if (this._chart._range) {
-      combineLatest([
-        merge(
-          getElementRefStream<HTMLDivElement>(
-            this._chart._range._stateChanges,
-            this._destroy$,
-            this._chart._range._rangeElementRef,
-            this._zone,
-          ),
-          this._dragHandle$.pipe(
-            getElementRef(this._chart._range._rangeElementRef),
-          ),
+      merge(
+        getElementRefStream<HTMLDivElement>(
+          this._chart._range._stateChanges,
+          this._destroy$,
+          this._chart._range._rangeElementRef,
+          this._zone,
         ),
+        this._dragHandle$.pipe(
+          getElementRef(this._chart._range._rangeElementRef),
+        ),
+      )
+        .pipe(
+          throttleTime(0, animationFrameScheduler),
+          takeUntil(this._destroy$),
+        )
+        .subscribe(ref => {
+          if (this._chart._range && this._chart._range._overlayTemplate) {
+            this._updateOrCreateOverlay(this._chart._range, ref, undefined);
+          }
+        });
+
+      // update the overlay position if an overlay exists
+      // and the viewport size changes
+      combineLatest([
+        getElementRef<HTMLDivElement>(this._chart._range._rangeElementRef),
```

Comment:
this does not work out `getElementRef` returns not an observable it is a pipeable custom operator function.